src/openlcb/StreamDefs.hxx

```diff
@@ -54,7 +54,10 @@ struct StreamDefs
     {
         FLAG_CARRIES_ID = 0x01,
         FLAG_REJECT_OUT_OF_ORDER = 0x02,
-        FLAG_PERMANENT_ERROR = 0x40,
+        /// @todo synchronize the draft text in the standards repository with
```

Comment:
Did you file an issue in the OpenLCB documents repo?

Reply:
https://github.com/openlcb/documents/issues/56